pgcli/pgexecute.py

```diff
@@ -378,6 +379,13 @@ def run(
 
             try:
                 if pgspecial:
+                    # \G is treated specially since we have to set the expanded output.
+                    if sql.endswith("\\G"):
```

Comment:
no big deal, but why don't we use raw string here too to avoid escaping?

Reply:
Good question. I just copy pasted the relevant bits from mycli. For whatever reason we didn't use raw strings in mycli.

 :smile: